ush/retrieve_data.py

```diff
@@ -946,25 +1012,29 @@ def parse_args(argv):
         one fhr will be processed.  If 2 or 3 arguments, a sequence \
         of forecast hours [start, stop, [increment]] will be \
         processed.  If more than 3 arguments, the list is processed \
-        as-is.",
+        as-is. default=[0]",
         nargs="+",
-        required=True,
+        required=False,                    # relaxed this arg option, and set a default value when not used
         type=int,
     )
     parser.add_argument(
         "--output_path",
         help="Path to a location on disk. Path is expected to exist.",
-        required=True,
+        required=False,                    # relaxed this arg option to allow for config setting from "disk" stores
         type=os.path.abspath,
     )
     parser.add_argument(
         "--ics_or_lbcs",
-        choices=("ICS", "LBCS"),
+        choices=("ICS", "LBCS", "None"),   # Added 'None' for generic packages copied from disk or remote
         help="Flag for whether ICS or LBCS.",
-        required=True,
+        required=False,                    # relaxed this arg option, to enable generic packages
```

Comment:
Same situation as for `anl_or_fcst` above.

Reply:
returned to True

Reply:
That doesn't show up here on the GitHub branch.

Reply:
@christinaholtNOAA should this be required? fetching obs. or fix files does not fall under LBC or IC